helm/cilium/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 0.25.1
+version: 0.0.0-dev
```

Comment:
from my POV having something in the repo that is not what we publish is still a problem and it will only bring confusion.

I suggest to ignore this file altogether for now and add automation in a future PR. The ignore should be added in vendir and in the sync.sh to avoid creating the diff.

Reply:
I agree with @mcharriere in this one.

Reply:
Restored.